test/test_shape_ops.py

```diff
@@ -378,21 +378,31 @@ def test_flip(self, device):
             self.assertEqual(size, list(data.flip(ds).size()))
 
         # test rectangular case
-        data = torch.tensor([1, 2, 3, 4, 5, 6]).view(2, 3).to(device)
-        flip0_result = torch.tensor([[4, 5, 6], [1, 2, 3]]).to(device)
-        flip1_result = torch.tensor([[3, 2, 1], [6, 5, 4]]).to(device)
+        data = torch.tensor([1, 2, 3, 4, 5, 6], device=device).view(2, 3)
```

Comment:
why do we need separate `test_flip`? Are there any tests here that are not covered by OpInfo tests?

Reply:
There are, unfortunately. Maybe in the future OpInfos will cover everything here.

Reply:
The Generic OpInfo tests do not verify the expected output of the operator against some reference.
That happens for UnaryUfuncInfo and others but not the generic one.

Thus the above tests are not covered.

Generic OpInfo tests cover basic behaviour of the operator like
* Raising an error for unsupported dtypes.
* Not raising an error for supported dtypes.
* Tests operators for consistency between JIT and eager
* gradcheck and gradgradcheck
* Verifying that out= output is same compared to when out is not passed (for applicable operators)

Reference: https://github.com/pytorch/pytorch/blob/master/test/test_ops.py